src/applications/vaos/new-appointment/redux/reducer.js

```diff
@@ -340,6 +341,36 @@ export default function formReducer(state = initialState, action) {
           vaParent,
         };
       } else {
+        if (
+          action.sortMethod === FACILITY_SORT_METHODS.DISTANCE_FROM_RESIDENTIAL
+        ) {
+          const residentialCoordinates = action.residentialCoordinates;
+          typeOfCareFacilities = typeOfCareFacilities
+            .map(facility => {
+              const distanceFromResidentialAddress = distanceBetween(
+                residentialCoordinates.latitude,
+                residentialCoordinates.longitude,
+                facility.position.latitude,
+                facility.position.longitude,
+              );
+
+              return {
+                ...facility,
+                legacyVAR: {
+                  ...facility.legacyVAR,
+                  distanceFromResidentialAddress,
+                },
+              };
+            })
+            .sort(
+              (a, b) =>
+                a.legacyVAR.distanceFromResidentialAddress <
```

Comment:
I think we need to wrap these distances in `parseFloat`. `distanceBetween` returns a string.

Reply:
Gotcha. I added the parseFloat to `distanceBetween` since I can't imagine any scenario where we'd want it as a string.